orc8r/cloud/go/obsidian/swagger/spec_servicer.go

```diff
@@ -0,0 +1,48 @@
+/*
+ Copyright 2020 The Magma Authors.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package swagger
+
+import (
+	"context"
+	"io/ioutil"
+
+	"magma/orc8r/cloud/go/obsidian/swagger/protos"
+	"magma/orc8r/lib/go/service/config"
+
+	"github.com/golang/glog"
+)
+
+type specServicer struct {
+	spec string
+}
+
+func NewSpecServicerFromFile(service string) protos.SwaggerSpecServer {
+	path := config.GetSpecPath(service)
+	data, err := ioutil.ReadFile(path)
+	if err != nil {
+		// We are swallowing this error because a singular service spec failure
```

Comment:
Nit: I don't like using "we" language in the codebase unless it's the only natural way to communicate the point. In this case, you can just say something like "Swallow ReadFile error to prevent ..."